install_config/storage_examples/ceph_rbd_dynamic_example.adoc

```diff
@@ -0,0 +1,313 @@
+.Ceph Secret Definition
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-secret
+  namespace: kube-system
+data:
+  key: QVFBOFF2SlZheUJQRVJBQWgvS2cwT1laQUhPQno3akZwekxxdGc9PQ== <1>
+type:
+  kubernetes.io/rbd <2>
+----
+<1> This base64 key is generated on one of the Ceph *MON* nodes using the `ceph auth
+get-key client.admin | base64` command, then copying the output and pasting it
+as the secret key's value.
+<2> This is required for Ceph RBD to work with dynamic provisioning.
+
+. Save the secret definition to a file, for example *_ceph-secret.yaml_*,
+then create the secret:
++
+----
+$ oc create -f ceph-secret.yaml
+secret "ceph-secret" created
+----
+
+. Verify that the secret was created:
++
+----
+# oc get secret ceph-secret
+NAME          TYPE                DATA      AGE
+ceph-secret   kubernetes.io/rbd   1         5d
+----
+
+[[ceph-rbd-dynamic-example-creating-the-ceph-user-secret]]
+== Creating the Ceph User Secret
+
+. Run the `ceph auth get-key` command on a Ceph *MON* node to display the key
+value for the *client.kube* user:
++
+.Ceph Secret Definition
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ceph-user-secret
+  namespace: default
+data:
+  key: QVFCbEV4OVpmaGJtQ0JBQW55d2Z0NHZtcS96cE42SW1JVUQvekE9PQ== <1>
+type:
+  kubernetes.io/rbd <2>
+----
+<1> This base64 key is generated on one of the Ceph *MON* nodes using the `ceph auth
+get-key client.kube | base64` command, then copying the output and pasting it as
+the secret key's value.
+<2> This is required for Ceph RBD to work with dynamic provisioning.
+
+. Save the secret definition to a file, for example *_ceph-user-secret.yaml_*,
+then create the secret:
++
+----
+$ oc create -f ceph-user-secret.yaml
+secret "ceph-user-secret" created
+----
+
+. Verify that the secret was created:
++
+----
+# oc get secret ceph-user-secret
+NAME               TYPE                DATA      AGE
+ceph-user-secret   kubernetes.io/rbd   1         5d
+----
+
+[[ceph-rbd-dynamic-example-create-storage-class]]
+=== Ceph RBD Dynamic Storage Class
+
+. Create a storage class to for your dynamic volumes.
++
+.ceph-storageclass.yaml
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1beta1
+kind: StorageClass
+metadata:
+  name: dynamic
+  annotations:
+     storageclass.beta.kubernetes.io/is-default-class: "true"
+provisioner: kubernetes.io/rbd
+parameters:
+  monitors: 192.168.1.11:6789,192.168.1.12:6789,192.168.1.13:6789  <1>
+  adminId: admin  <2>
+  adminSecretName: ceph-secret  <3>
+  adminSecretNamespace: kube-system  <4>
+  pool: kube  <5>
+  userId: kube  <6>
+  userSecretName: ceph-secret-user  <7>
+----
+<1> Ceph monitors, comma delimited. It is required.
+<2> Ceph client ID that is capable of creating images in the pool. Default is `admin`.
+<3> Secret Name for `adminId`. It is required. The provided secret must have type `kubernetes.io/rbd`.
+<4> The namespace for `adminSecret`. Default is `default`.
+<5> Ceph RBD pool. Default is `rbd`.
```

Comment:
@mburke5678 Good catch. I'll emphasize that again here. 
@donnydavis Are there any other recommended values besides `kube`?